server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -642,6 +642,62 @@ public class ShuffleServerConf extends RssBaseConf {
           .defaultValue(10 * 60L)
           .withDescription("The storage remove resource operation timeout.");
 
+  public static final ConfigOption<Boolean> SERVER_MERGE_ENABLE =
+      ConfigOptions.key("rss.server.merge.enable")
+          .booleanType()
+          .defaultValue(false)
+          .withDescription("Whether to enable remote merge");
+
+  public static final ConfigOption<Integer> SERVER_MERGE_THREAD_POOL_SIZE =
+      ConfigOptions.key("rss.server.merge.threadPool.size")
+          .intType()
+          .defaultValue(10)
+          .withDescription("thread pool for merge");
+
+  public static final ConfigOption<Integer> SERVER_MERGE_THREAD_POOL_QUEUE_SIZE =
```

Comment:
We would better to follow the style of Spark. The last segment use camel style.
`rss.server.merge.threadPoolSize`

Reply:
> We would better to follow the style of Spark. The last segment use camel style. `rss.server.merge.threadPoolQueueSize`

@jerqi In fact, I see many style in ShuffleServerConf.java. 
I understand when you say the style is "rss.server.merge.{last camel style word}", is that so?

Do I need change all new config? For example: rss.server.merge.class.loader.jars.path -> rss.server.merge.classLoaderJarsPath, rss.server.merge.block.ring.buffer.size -> rss.server.merge.blockRingBufferSize. rss.server.merge.cache.merged.block.max.sleep.ms -> rss.server.merge.cacheMergedBlockMaxSleepMs.

Reply:
Yes, let use gradually change the config style to this kind.

Reply:
> Yes, let use gradually change the config style to this kind.

OK, I have change the style.